crates/ruma-common/src/api/metadata.rs

```diff
@@ -124,6 +124,13 @@ impl Metadata {
 
         Ok(res)
     }
+
+    // Used for generated `#[test]`s
+    #[doc(hidden)]
+    pub fn _path_parameters(&self) -> Vec<&'static str> {
```

Comment:
Why not make this public, actually?

It would have low value for developers, but I think it'd be good to do it anyways, as it would be convenient for when they do want to grab this

Reply:
I'd prefer to default to hidden until there is an actual use case. It's easy to make public later.